bin/setup

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+project=mynamespace
+
+# Check that minishift is installed
+minishift=$(which minishift)
+if [ -z "$minishift" ];
+then
+	echo "Minishift not found, see https://github.com/minishift/minishift#getting-started for installation instructions"
+	exit 1
+fi
+
+# Ensure minishift is running
+is_running=$(minishift status | grep Minishift | awk '{print $2}')
+if [ "$is_running" != "Running" ];
+then
+	echo "You must start minishift"
+	exit 1
+fi
+
+# Make sure path to the oc binary is on our PATH
+eval $(minishift oc-env)
+
+# Login as admin
+res=$(oc login -u system:admin)
+
+res=$(oc describe scc anyuid | grep Users | awk '{print $2}' | grep miq-anyuid)
```

Comment:
This will return the result for all namespaces. I think this should be `grep "$project:miq-anyuid"`

Reply:
Also is it better to use `grep -q` here?

Reply:
:+1: switching to that and checking the return code